scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/rmproxies/SingleConnectionRMProxiesManager.java

```diff
@@ -52,9 +52,9 @@ synchronized public URI getRmUrl() {
     }
 
     @Override
-    synchronized public void rebindRMProxiesManager(URI rmURI) throws RMException, RMProxyCreationException {
+    synchronized public boolean rebindRMProxiesManager(URI rmURI) throws RMException, RMProxyCreationException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the redundant '!unknownSymbol!' thrown exception declaration(s). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)